### DISCUSSION TITLE
Pod-airbyte-bootloader-error

### DISCUSSION THREAD

**Discussion by nikhilnicky1** [airbytehq/airbyte#45177] [Deployment] [answered]
Here is my values.yaml file which connects to RDS and S3 in storage, by default its creating airbyte-db-0 pod and iam getting pod-airbyte-bootloader error, when i checked the logs its trying to connect localhost db. can anyone please help am i missing something in the configuration?? iam new to k8s.

[New Text Document.txt](https://github.com/user-attachments/files/16898623/New.Text.Document.txt)
![image](https://github.com/user-attachments/assets/57eb6fbf-eba4-4b04-a90c-816ddd18c6dd)

**Comment by NAjustin** [comment]
@nikhilnicky1 The `postgresql` and `minio` config keys need to be at the top level (same as `global`)—not indented like in your current config. Also, I don't think the `minio.enabled` flag is needed anymore (IIRC it's not inferred by `global.storage.type`).

Otherwise it looks about right to me, so maybe try that first.

**Reply by nikhilnicky1** [reply]
![image](https://github.com/user-attachments/assets/6de9797c-ac11-4c50-9944-924eb66e2267)
Thankyou for the response, here is the updated file but the same output iam getting the same error. Iam attaching the logs please check.
[logs.txt](https://github.com/user-attachments/files/16898954/logs.txt)

**Reply by NAjustin** [reply, marked as answer]
@nikhilnicky1 The localhost errors you're seeing are a red herring . . . that's the stats collector, not Postgres. The line that actually diagnoses the issue is this one:
```
Caused by: java.net.UnknownHostException: postgres.cluster-czcu8isw0xzm.us-east-1.rds.amazonaws.com
```

It would seem that the host machine can't resolve that name, likely because it's an internal DNS record for AWS and the host machine doesn't have that zone available (it's definitely not a public hostname).

I would start by making sure that you have the right connection details, and that the hose machine for Airbyte can resolve the hostname (you can always test this using `nslookup`/`dig`/`ping`). Some hostnames in AWS are regional, meaning they'll only resolve for instances in the same region and not outside. In other cases, you could have to do additional configuration of the VPC settings or access rules.

If you're ever not sure about connectivity, you can always install the postgres client and try to connect directly—sometimes you'll get a more helpful error message. (Or if nothing else you can rule out Airbyte as the issue and know where to better focus your efforts.)
